AjaxFetch/js/show.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * show users
+ * @param {string} parent
+ */
+function user(parent) {
+  $.each(window.values.users, (userId, user) => {
+    let posts = $(`<div style="display: none"></div>`);
+    let albums = $(`<div style="display: none"></div>`);
+    let userName = $(`<h2 class="well">${user.name}</h2>`);
```

Comment:
Try to use this method of creating elements with jQuery: 
```let userName = $(`<h2/>').addClass('well').text(user.name);```
This is better way for understanding your code.